local-proxy/package.json

```diff
@@ -29,6 +30,7 @@
     "@types/nanoid": "^2.0.0",
     "@types/node": "^10.14.9",
     "@types/rimraf": "^2.0.2",
+    "tslint": "~5.18.0",
```

Comment:
Note this rush peculiarity, it prefers tilde to caret, can you change to `rush add --caret --dev -p tslint`? Although, this will probably become irrelevant when we change to automatic updates.